### DISCUSSION TITLE
Binary Results Output to User-Specified Directory

### DISCUSSION THREAD

**Discussion by johodges** [firemodels/fds#12391] [General]
We recently added the ability for users to specify a RESULTS_DIR on the DUMP line which will be used by FDS to store binary result files. I know pyfdstools will need some updates to support this feature, and I expect fds2ascii will as well. What other areas of the codebase/utilities will need to be updated to support?

**Comment by rmcdermo** [comment]
Given that this is an option, I can't think of anything at the moment.  But we should probably discuss at a developer meeting soon and get opinions from @bwklein and @emanuelegissi .
